### PR TITLE
adds ability to create static pages

### PR DESCRIPTION
**Pull request by Danwhy**
### Merge after https://github.com/LDMW/cms/pull/43
ref #367
Pulls through static pages from cms
Separates out crisis page as it has a custom design that will not match other static pages

### CODE REVIEW

File: web/controllers/crisis_controller.ex [hunk 1]
```diff
@@ -0,0 +1,21 @@
+defmodule App.CrisisController do
+  use App.Web, :controller
+
+  import Ecto.Query, only: [from: 2]
```
Comment (shouston3) [inline web/controllers/crisis_controller.ex]: We can remove this line as it is already imported by `use App.Web, :controller` [here](https://github.com/LDMW/app/blob/master/web/web.ex#L35)

File: web/controllers/static_controller.ex [hunk 2]
```diff
@@ -1,16 +1,15 @@
-defmodule App.InfoController do
+defmodule App.StaticController do
   use App.Web, :controller
 
   import Ecto.Query, only: [from: 2]
```
Comment (shouston3) [inline web/controllers/static_controller.ex]: We can remove this line too

**Review by samhstn — COMMENTED** [review]
Two minor points

**Review by SimonLab — APPROVED** [review]
LGTM

File: web/controllers/crisis_controller.ex (hunk 1 above)
Comment (shouston3) [inline web/controllers/crisis_controller.ex]: This was actually me, oops. We can address this in a separate pr

**Pull request merged**